### PR TITLE
[12.0][FIX]analytic_category dependencies

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**

### CODE REVIEW

File: analytic_category/__manifest__.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "version": "12.0.1.0.0",
     "author":  "Eficent",
     "category": "Analytic",
-    "depends": ["analytic"],
+    "depends": ["analytic", "account"],
```
Comment (MiquelRForgeFlow) [inline analytic_category/__manifest__.py]:
```suggestion
    "depends": ["account"],
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: analytic_category/__manifest__.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply analytic_category/__manifest__.py]: analytic does not depend on account

Comment (MiquelRForgeFlow) [reply analytic_category/__manifest__.py]: it's the contrary, `account` depends of `analytic`, so putting `account` here already implies `analytic`.

Comment (AaronHForgeFlow) [reply analytic_category/__manifest__.py]: I believe you

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]
SQUASH

**Review by MiquelRForgeFlow — COMMENTED** [review]
:+1:

**Pull request merged**